Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -615,6 +617,33 @@ - (nullable NSString *)currentScreenNameForSessionReplay
                .firstObject;
 }
 
+- (void)showMaskPreview:(CGFloat)opacity
+{
+    if ([SentryDependencyContainer.sharedInstance.crashWrapper isBeingTraced] == NO) {
+        return;
+    }
+
+    UIWindow *window = SentryDependencyContainer.sharedInstance.application.windows.firstObject;
+    if (window == nil) {
+        SENTRY_LOG_WARN(@"There is no UIWindow available to display preview");
+        return;
+    }
+
+    if (_previewView == nil) {
+        _previewView = [[SentryMaskingPreviewView alloc] initWithRedactOptions:_replayOptions];
+    }
+
+    _previewView.opacity = opacity;
+    [_previewView setFrame:window.bounds];
+    [window addSubview:_previewView];
```

Comment:
`m`: This will crash when somebody calls `SentrySDK.replay.showMaskPreview(0.5)` from a BG thread. I think we can ignore this scenario by clearly pointing out in the code docs that this method must be called from the main thread.

Reply:
It will not crash, users get a warning from UIKIt that they are handling ui in a background thread.